lib/Image/TIFF.pm

```diff
@@ -18,7 +18,6 @@ my @types = (
   [ "RATIONAL",  "N2", 8],
   [ "SBYTE",     "c1", 1],
   [ "UNDEFINED", "a1", 1],
-  [ "BINARY",    "a1", 1],  # treat binary data as UNDEFINED	
```

Comment:
Order of types after `BINARY` was shifted from those defined in TIFF specification <https://www.itu.int/itudoc/itu-t/com16/tiff-fx/docs/tiff6.pdf#page=16>.